verification/curator-service/ui/src/components/bulk-case-form-fields/FileUploadProgress.tsx

```diff
@@ -0,0 +1,33 @@
+import CircularProgress, {
+    CircularProgressProps,
+} from '@material-ui/core/CircularProgress';
+
+import Box from '@material-ui/core/Box';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+export default function FileUploadProgress(
```

Comment:
Nit: this isn't specific to Files so we could name it just UploadProgress in case it's needed somewhere else in the future.

Reply:
SG, done.